app/assets/stylesheets/utils/_custom.scss

```diff
@@ -37,3 +37,16 @@
 .govuk-link:link {
     font-weight: normal;
 }
+
+.account-status-header {
+    line-height: 1;
+    text-transform: uppercase;
+    padding: 5px 8px 4px;
+    outline: 2px solid rgba(0, 0, 0, 0);
+    outline-offset: -2px;
+    font-weight: 700;
+    font-size: 14px;
```

Comment:
You have two `font-size`. I would just remove them since new `govuk-tag` class has a font-size anyway

Reply:
I have removed this redundant one but has kept font-size: 1rem; as if I remove this the text spreads over two lines for status "PENDING OPENING ON CDS"